dymos/phases/options.py

```diff
@@ -318,6 +318,11 @@ def __init__(self, read_only=False):
                      desc='setting to control if segment collocation defects are '
                           'solved with a newton solver')
 
+        self.declare('time_direction', default='forward', values=('forward', 'backward'),
+                     desc='Whether the numerical propagation occurs forward or backward '
+                          'in time for this state.  This poses restrictions on whether '
+                          'states can have fixed or connected initial and final values.')
```

Comment:
What are the restrictions? Looks like you can only fix the end you start on, so for forward: fix_final is False, and backward: fix_initial is False.  That guideline is small enough to fit here, i think.  Will also need to eventually check this.  BTW, what do these options do for non RK phases?

Reply:
If an end is fixed or connected, the propagation must start on that end.  I don't think the current formulation can solve when both ends are fixed/connected, but I haven't verified that yet.  Theres a pretty complicated logic table and I don't think it fits here, but we can put it in the docs.